.github/dependabot.yml

```diff
@@ -4,3 +4,11 @@ updates:
     directory: "/"
```

Comment:
```suggestion
    directory: "/"
    versioning-strategy: increase
```

Reply:
this failed validation, it seems this is not available for the GH ecosystem

Reply:
https://github.com/prebid/Prebid.js/runs/46425081234